### PR TITLE
Convert RecyclerViewIndicatorDotDecor to Kotlin

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        }
+        drawActiveDot(canvas, indicatorStartX, indicatorPositionY, activePosition)
+    }
+
+    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
+        super.getItemOffsets(outRect, view, parent, state)
+        outRect.bottom = indicatorHeight
+    }
+
+    private fun drawInactiveDots(canvas: Canvas,
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt:71]: can we have the parameters in the same line please?

File: app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                                 itemCount: Int) {
+        val itemWidth = radius * 2 + indicatorItemPadding
+        var start = indicatorStartX + radius
+        for (i in 0 until itemCount) {
+            canvas.drawCircle(start, indicatorPositionY, radius, inactivePaint)
+            start += itemWidth
+        }
+    }
+
+    private fun drawActiveDot(canvas: Canvas,
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt:83]: Same

**Review by sharvaniharan — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/feed/news/RecyclerViewIndicatorDotDecor.kt]: Done!

**Comment by cooltey** [comment]
@sharvaniharan This is done and ready to be reviewed and merged.

**Pull request merged**